hubs.yaml

```diff
@@ -13,7 +13,8 @@ clusters:
     hubs:
       - name: staging
         cluster: 2i2c
-        domain: staging.pilot.2i2c.cloud
+        domain:
```

Comment:
This should be optionally a list or string, since most hubs will not have more than one domain